.github/workflows/ci_performance_metrics_monitoring.yml

```diff
@@ -0,0 +1,89 @@
+name: "[CI] Performance Metrics monitoring"
```

Comment:
This could probably be DRY'ed up with the composite actions from #8677, but I'm not so sure (for sure we should rename it, as it has rspec in the name/description/filename, brakeman should be dropped, also there isn't a module name on this case...

Reply:
No with the current composited action, because it launches a test app, and I'm creating a dev app.

Maybe I can extract the setup ruby and setup node steps to another composite action, let me try!

Reply:
Ok, let me know if you've arrived to any conclusion. My comment was as a "nice to have/wouldn't it be cool if..." but I can accept the PR as is now 😄 .